test/components/canBeCreatedWithItems.ts

```diff
@@ -0,0 +1,55 @@
+import { IPokemon } from "../../src/components/Battles";
+import { INewPokemon } from "../../src/components/constants/Pokemon";
+import { FullScreenPokemon } from "../../src/FullScreenPokemon";
+import { it } from "../main";
+import { stubBlankGame } from "../utils/fakes";
+
+const pokemonTitle: string[] = "CHARMANDER".split("");
+
+it("Pokemon can be given an item when created", (): void => {
+    // Arrange
+    const fsp: FullScreenPokemon = stubBlankGame();
+    // Act
+    const pokemon: IPokemon = fsp.equations.newPokemon({
+        level: 1,
+        title: pokemonTitle,
+        item: "Potion".split("")
+    });
+
+    // Assert
+    chai.expect(pokemon.item).to.deep.equal("Potion".split(""));
+});
+
+it("RandomHeldItemMod makes Pokemon spawn with random item when the generated probability is valid", (): void => {
```

Comment:
`RandomHeldItemMod` tests should be in its own file since it's its own component.